### ISSUE TITLE
Training is used triplet loss function? or Center loss function and softmax loss function?

### ISSUE THREAD

**Issue by sy0209** [davidsandberg/facenet#471]

**Comment by allenxcp** [comment]
center loss
